src/core/config.ts

```diff
@@ -2,7 +2,7 @@ import { Config } from "@clarity-types/core";
 
 let config: Config = {
     projectId: null,
-    longtask: 30, // 30 milliseconds
+    yield: 30, // 30 milliseconds
```

Comment:
>yield [](start = 4, length = 5)

Might want to avoid keyboards. yieldTime?

Reply:
Good point. Let me go back to longtask as the setting name.